presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/StatisticsBuilders.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata.statistics;
+
+import com.facebook.presto.orc.ColumnWriterOptions;
+import com.facebook.presto.orc.metadata.OrcType;
+
+import java.util.function.Supplier;
+
+import static java.util.Objects.requireNonNull;
+
+public class StatisticsBuilders
+{
+    private StatisticsBuilders() {}
+
+    public static Supplier<StatisticsBuilder> createStatisticsBuilderSupplier(OrcType orcType, ColumnWriterOptions columnWriterOptions)
+    {
+        requireNonNull(orcType, "orcType is null");
+        requireNonNull(columnWriterOptions, "columnWriterOptions is null");
+
+        switch (orcType.getOrcTypeKind()) {
+            case BYTE:
+                return CountStatisticsBuilder::new;
```

Comment:
IIRC, This is different from BBIO, it typically uses Integer. But we can consolidate them later and byte is not a widely used type.

Reply:
Byte in Presto uses generic stats, in BBIO it's integer stats. Yeah, we would need to fix this. 
I need this to make sure that OrcTester test cases pass, otherwise it will fail on the stats validation.